tests/check-http-method.spec.js

```diff
@@ -0,0 +1,16 @@
+const HTTP_METHODS = require('../lib/http-methods');
+const checkHttpMethod = require('../lib/check-http-method');
+
+describe('checkHttpMethod function ->', () => {
+    HTTP_METHODS.forEach(method => {
+        it(`should treat HTTP method "${method}" as a correct method`, () => {
+            const result = checkHttpMethod(method);
+            expect(result).toBe(true);
+        });
+    });
+
+    it(`should be able to detect an incorrect http method`, () => {
```

Comment:
Just one last thing.
I guess we don't need template strings in this case since we don't use any variables.

Reply:
Sure.